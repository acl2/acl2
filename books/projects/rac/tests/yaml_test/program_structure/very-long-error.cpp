#include <array>
using namespace std;

// RAC begin

// clang-format off

int foo() {

  array<int, 1> a;

  return 1
    + 2
    + 3
    + 4
    + 5
    + 6
    + 7
    + 7
    + a;
}
