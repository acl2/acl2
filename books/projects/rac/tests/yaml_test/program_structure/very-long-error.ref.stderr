very-long-error.cpp:12-20 (9-7): Cannot apply `+` to arguments of type int and int[1]
(Too much context, show only the 5 first lines)
 12 |   return 1
    |          ^^
 13 |     + 2
    | ^^^^^^^^
 14 |     + 3
    | ^^^^^^^^
 15 |     + 4
    | ^^^^^^^^
 16 |     + 5
    | ^^^^^^^^
 17 |     + 6
    | ^^^^^^^^
note: They should be both a primitive type (like an int or enum) or a register type.
